tallycash/extension
# Fix Quickswap Integration

Closes https://github.com/tallycash/extension/issues/1785

This PR:
- Explicitly attaches the jsonrpc version to messages we send through the provider-bridge
- Caches the window.ethereum walletrouter Proxy so we don't run into edge cases with strict equality and `window.ethereum`.
- Adds some hard coded paths for quickswap while we figure out the best way forward with handling dapps that expect permissions without being granted them

To Test:
- connect to quickswap.exchange
- Ensure you're using their new UI (you might need to clear cache)
- Connect and swap
